services/navigation/builtin/builtin.go

```diff
@@ -280,9 +280,15 @@ func (svc *builtIn) SetMode(ctx context.Context, mode navigation.Mode, extra map
 	cancelCtx, cancelFunc := context.WithCancel(context.Background())
 	svc.wholeServiceCancelFunc = cancelFunc
 	svc.mode = mode
-	if svc.mode == navigation.ModeWaypoint {
+
+	switch svc.mode {
+	case navigation.ModeManual:
```

Comment:
is there no action here because `svc.stopWaypointMode()` puts navigation in ManualMode?

Reply:
Manual mode means navigation does nothing. `stopWaypointMode()` is run every time. I am renaming this to `stopActiveMode()` to cover any and all navigation modes that have an active element such as waypoint and now explore.